.styleci.yml

```diff
@@ -0,0 +1,11 @@
+preset: symfony
+
+finder:
+    exclude:
+        - "spec"
+        - "tests"
+    path:
+        - "src"
```

Comment:
we follow the symfony bundle best practices here, so these paths are different. the code path is ., exclude is Tests. there is no spec folder.